language-support/hs/bindings/src/DA/Ledger/Jwt.hs

```diff
@@ -4,21 +4,32 @@
 module DA.Ledger.Jwt(
   Jwt,
   tryCreateFromString,
-  toString
+  tryCreateFromBearerString,
+  toBearerString
   ) where
 
 import Data.List.Extra (splitOn)
 import Data.String.Utils (strip)
 
 newtype Jwt = Jwt { toString :: String }
 
+toBearerString :: Jwt -> String
+toBearerString s = "Bearer " <> toString s
+
+-- | catch easy user errors before sending the token to the ledger
+tryCreateFromBearerString :: String -> Either String Jwt
+tryCreateFromBearerString s0 = do
+  let s = strip s0
+  case splitOn " " s of
+    ["Bearer",s] -> tryCreateFromString s
```

Comment:
Rather than stripping bearer here only to prepend it later, we should pass it along verbatim. The change to the format in the integration tests is correct, so this is only about how we process the contents of that file.

Reply:
ok.
so we should make no attempt to validate the token before passing in on to the ledger?

Reply:
we discussed, the answer is "yes" - no validation
code fixed